README.md

```diff
@@ -65,19 +65,32 @@ render() {
 
 ### AutoRotatingCarousel Properties
 
-|Name            |Type        |Default     |Description
-|----------------|------------|------------|--------------------------------
-|autoplay        | `bool`     | `true`     | If `false`, the auto play behavior is disabled.
-|contentStyle    | `object`   |            | Override the inline-styles of the content container.
-|interval        | `integer`  | `3000`     | Delay between auto play transitions (in ms).
-|label           | `string`   |            | Button text. If not supplied, the button will be hidden.
-|landscape       | `bool`     |            | If `true`, slide will adjust content for wide mobile screens.
-|mobile          | `bool`     | `false`    | If `true`, the screen width and height is filled.
-|open            | `bool`     | `false`    | Controls whether the AutoRotatingCarousel is opened or not.
-|onChange        | `function` |            | Fired when the index changed. Returns current index.
-|onRequestClose  | `function` |            | Fired when the gray background of the popup is pressed when it is open.
-|onStart         | `function` |            | Fired when the user clicks the getting started button.
-|style           | `object`   |            | Override the inline-styles of the root component.
+|Name                 |Type        |Default     |Description
+|---------------------|------------|------------|--------------------------------
+|arrowIcon            | `object`   |            | Override the inline-styles of the arrow icon.
+|arrowIconButtonStyle | `object`   |            | Override the inline-styles of the arrow icon button.
+|arrowLeftStyle       | `object`   |            | Override the inline-styles of the left arrow.
+|arrowRightStyle      | `object`   |            | Override the inline-styles of the right arrow.
+|autoplay             | `bool`     | `true`     | If `false`, the auto play behavior is disabled.
+|carouselContainer    | `object`   |            | Override the inline-styles of the carousel container.
+|carouselStyle        | `object`   |            | Override the inline-styles of the carousel.
+|carouselWrapperStyle | `object`   |            | Override the inline-styles of the carousel wrapper.
```

Comment:
So… We have a carousel container, a carousel style and a carousel wrapper… uhm… :sweat_smile: Does the user really need to override all of them?

Reply:
Here is my thought: though the styles for carousel container, wrapper have been added, exposing the inline style to devs may give more freedom to take use of this carousel. Also this is not a totally overwriting approach, it would only modify the styles devs need to change and keep others the same, so I think it would not change too much. But that all depend on you. If want to hide these styles I would delete these change

Reply:
My idea is exposing all the styles defined in desktopStyles/mobileStyles to developers. If you think that would change the Material-UI style, please let me know which inline styles you think are okay to expose them.

Reply:
Okay, fine. :+1: Let's expose all the styles, but please don't differentiate between `...LandscapeStyle` and `...portraitStyle`. The user can do that as the user also explicitly sets `landscape` so that they can also change the styles instead of providing both as two props.